workflow-templates/dependency-submission/action.yml

```
@@ -0,0 +1,24 @@
+name: dependency submission
+description: generate a dependency graph
+
+inputs:
+  jdk-version:
+    description: "The JDK version to use"
+    default: ""
+    required: true
+  jdk-distribution:
+    description: "The JDK distribution to use"
+    default: ""
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
```

Add a blank line here.

```suggestion
    - uses: actions/checkout@v4

```